libs/dh/market-participant/actors/feature-organizations/src/lib/dh-organizations-overview.component.ts

```diff
@@ -99,11 +98,10 @@ export class DhOrganizationsOverviewComponent {
   isLoading = this.getOrganizationsQuery.loading;
   hasError = computed(() => this.getOrganizationsQuery.error !== undefined);
 
-  navigate(id: string | null) {
-    this.router.navigate([], {
+  navigate(id: string | undefined) {
+    this.id.set(id);
+    this.router.navigate(['details', id], {
       relativeTo: this.route,
-      queryParams: { id },
-      queryParamsHandling: 'merge',
     });
```

Comment:
suggestion: a similar method that handles the closing of the drawer should be inside the `dh-organization-details.component.ts` as well. Right now the url doesn't update when the drawer is closed.
Then this class should also look at the route params in order to set the correct row as active, and clearing it when the drawer closes.